optimistic-transaction.md

```diff
@@ -7,35 +7,36 @@ aliases: ['/docs/dev/reference/transactions/transaction-optimistic/','/docs/dev/
 
 # TiDB Optimistic Transaction Model
 
-This document introduces the principles of TiDB's optimistic transaction model. This document assumes that you have a basic understanding of [TiDB architecture](/architecture.md), [Percolator](https://ai.google/research/pubs/pub36726), and the [ACID](/glossary.md#acid) properties of transactions.
+This document introduces the principles of TiDB's optimistic transaction model and related features.
 
-In TiDB's optimistic transaction model, the two-phase commit begins right after the client executes the `COMMIT` statement. Therefore, the write-write conflict can be observed before the transactions are actually committed.
+TiDB uses the optimistic transaction model by default. In TiDB's optimistic transaction model, the two-phase commit begins only after the transaction is committed.
```

Comment:
The second sentence is not aligned with https://github.com/pingcap/docs-cn/pull/3171/files @ran-huang

Reply:
Updated.